### PR TITLE
Showcase Popup UI Fixes

### PR DESCRIPTION
**Pull request by Genoux**
- [x] Fix missing style on articles page
- [x] Fix showcase popup width (**On safari**)

### CODE REVIEW

**Comment by Genoux** [comment]
@afunnydev Can you help me figure out why @import `"partials/showcase-popup";` won't apply in articles-main.scss?

Issue can be found here -> https://deploy-preview-276--mystifying-allen-be5d31.netlify.app/articles/marketing-agencies-in-uk/

**Comment by afunnydev** [comment]
It is an `article` so it picks up the styling from `artical-main` if I am not mistaken. I don't see the partial in that SCSS file.

File: themes/inbeat/assets/scss/article-main.scss [hunk 1]
```diff
@@ -9,6 +9,7 @@
 @import "partials/footer";
 @import "partials/popup";
 @import "partials/banner.scss";
+@import "partials/showcase-popup";
```
Comment (Genoux) [inline themes/inbeat/assets/scss/article-main.scss:12]: @afunnydev Added here - Wrong file?

**Comment by afunnydev** [comment]
Sorry. You need to run the gulp task `sass-article`. The easiest way to do so is running `npm run watch-article` and then make a change in the SCSS to trigger the pipeline. There are other ways haha, pretty old setup!

**Review by afunnydev — APPROVED** [review]

**Comment by afunnydev** [comment]
LGTM

**Pull request merged**